adoc/entities.adoc

```diff
@@ -43,8 +44,7 @@
 :openscap: OpenSCAP
 :uarr:
 :ncc: Novell Customer Center
-:sp-ver: {sp-version}
-:smtool: {rmtool}
+:smtool: Subscription Management
```

Comment:
Should this be "Subscription Management Tool"?

Reply:
> Should this be "Subscription Management Tool"?

;)